### PR TITLE
Added BIO2021Q3

### PR DESCRIPTION
**Pull request by macaquedev**

### CODE REVIEW

File: content/problems/bio1/index.tsx [hunk 1]
```diff
@@ -203,6 +204,16 @@ export const bio1Problems: ProblemInfo[] = [
     solutionAuthors: ["Alex Pylypenko"],
     component: <BlockPalindromes />,
   },
+  {
+    difficulty: "Medium",
+    display: "Window Dressing",
+    original: "https://www.olympiad.org.uk/papers/2023/bio/bio23-exam.pdf",
```
Comment (myst-6) [inline content/problems/bio1/index.tsx:210]: this link is wrong

**Review by myst-6 — CHANGES_REQUESTED** [review]
editorial is all good but just the metadata is slightly wrong

File: content/problems/bio1/index.tsx (hunk 1 above)
Comment (macaquedev) [reply content/problems/bio1/index.tsx:210]: derp

**Review by AG-88301 — DISMISSED** [review]
You forgot to remove Vlad's code for dreaming spires

File: solutions/bio1/WindowDressing/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <SText>
+        There are many BFS problems in BIO Round 1, so it is
+        important to get confident implementing 
+        this algorithm quickly. A very similar problem 
+        is "Dreaming Spires", BIO 2023 R1 Q3.
+      </SText>
+      <SText>
+        This editorial was a brief explanation of BFS in the context of this problem. For a more rigorous coverage, visit<Link href="https://usaco.guide/silver/graph-traversal?lang=cpp">this USACO Guide page</Link>.
+      </SText>
+      <SCodeBlock path="dreamingspires/sol" />
```
Comment (AG-88301) [inline solutions/bio1/WindowDressing/index.tsx]: Kill this

Comment (AG-88301) [reply solutions/bio1/WindowDressing/index.tsx]: (Line 40)

**Review by AG-88301 — APPROVED** [review]
good

**Pull request merged**